### ISSUE TITLE
Link prediction

### ISSUE THREAD

**Issue by MrYaoH** [ckldan520/DDGCL#6]
Hi, author:
I notice that the code just be used to node classification. Could you please share the code about link prediction? Thank you very much!

**Comment by ckldan520** [comment]
The code for link prediction needs to be reviewed by the company before it can be uploaded, which takes about 1-2 months.
     There is little difference between link prediction and node classification codes. I have not modified the overall network structure, but there is a slight difference in data preprocessing. You can directly learn from the processing operation of link prediction in the TGN paper(https://github.com/twitter-research/tgn).  Specifically, function "get_data" (in utils.data_processing) is used as the division of training set and test set, and 'RandEdgeSampler' (in utils.utils) is used as the production of negative samples. It is suggested that if you are in a hurry and need to use it, you can directly modify it for link prediction.

**Comment by MrYaoH** [comment]
> ```
> The code for link prediction needs to be reviewed by the company before it can be uploaded, which takes about 1-2 months.
>  There is little difference between link prediction and node classification codes. I have not modified the overall network structure, but there is a slight difference in data preprocessing. You can directly learn from the processing operation of link prediction in the TGN paper(https://github.com/twitter-research/tgn).  Specifically, function "get_data" (in utils.data_processing) is used as the division of training set and test set, and 'RandEdgeSampler' (in utils.utils) is used as the production of negative samples. It is suggested that if you are in a hurry and need to use it, you can directly modify it for link prediction.
> ```

Thank you for your help！I will try it by myself.

**Comment by YuanchenBei** [comment]
hi, authors:
    The DDGCL is an excellent work! Have the link prediction code available now?
